bin/asd-prereq-checker.sh

```diff
@@ -11,6 +11,11 @@ SERVICE_MANAGEMENT_API=servicemanagement.googleapis.com
 
 PROJECT_ID=$(gcloud config get-value project)
 
+USER_IS_NOT_PROJECT_OWNER="{
+  'KnownIssueId': 'invalid_iam',
+  'Message': 'User needs have IAM Owner role for this project $PROJECT_ID.  https://console.cloud.google.com/iam-admin/iam?project=$PROJECT_ID '
```

Comment:
User needs to have

Reply:
User doesn't exactly have to have the "project owner" role.

Reply:
done